File: CS/Form1.cs
using System;
using System.Windows.Forms;
using System.Drawing.Imaging;
using DevExpress.XtraMap;
using System.Diagnostics;

namespace ExportSelectedItems {
    public partial class Form1 : Form {
        string mapPath = "Image.png";

        public Form1() {
            InitializeComponent();
            mapControl1.ExportMapItem += mapControl1_ExportMapItem;
        }

        private void mapControl1_ExportMapItem(object sender, ExportMapItemEventArgs e) {
            if (!e.IsSelected)
                e.Cancel = true;
        }

        private void simpleButton1_Click(object sender, EventArgs e) {
            mapControl1.ExportToImage(mapPath, ImageFormat.Png);
            Process.Start(mapPath);
        }
    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
Imports System.Drawing.Imaging
Imports DevExpress.XtraMap
Imports System.Diagnostics

Namespace ExportSelectedItems
	Partial Public Class Form1
		Inherits Form
		Private mapPath As String = "Image.png"

		Public Sub New()
			InitializeComponent()
			AddHandler mapControl1.ExportMapItem, AddressOf mapControl1_ExportMapItem
		End Sub

		Private Sub mapControl1_ExportMapItem(ByVal sender As Object, ByVal e As ExportMapItemEventArgs)
			If (Not e.IsSelected) Then
				e.Cancel = True
			End If
		End Sub

		Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			mapControl1.ExportToImage(mapPath, ImageFormat.Png)
			Process.Start(mapPath)
		End Sub
	End Class
End Namespace
